src/components/markdown/Tabs.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Tabs({ markdownFiles }) {
```

Comment:
Do you need to be passing down the entire markdown file?

Reply:
implemented change